internal/controllers/network/tests/import/02-cleanup.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+commands:
+- script: "! kubectl get network import --namespace $NAMESPACE"
```

Comment:
This should not be a `TestAssert` in this file, but a `TestStep`. See how https://github.com/k-orc/openstack-resource-controller/blob/main/internal/controllers/flavor/tests/import/02-cleanup.yaml does it.

Reply:
Done